__tests__/unit/lib/utils/childProcessUtils.test.js

```diff
@@ -70,6 +71,36 @@ describe('childProcessUtils', () => {
         }
       })
     })
+
+    describe('when stream emits error but no stderr output', () => {
+      it('throws an empty error', async () => {
+        // Arrange
+        expect.assertions(1)
+        const stream = new EventEmitter()
```

Comment:
**thought:** I wonder if it could be easy to isolate the generation of the stream in a test helper method

That way it would be easier to read the arrange part of those test.
Not sure it is worth it though as this part of the code may not change soon and is subject to be replaced by a future refactoring.

Something like this could be nice: 
```js
const arrangeStream = (data, error, isError) => {
  const getReadable = content => {
    return new Readable({
      read() {
        if (content) this.push(content)
        this.push(null)
      },
    })
  }
  const stream = new EventEmitter()
  stream.stdout = getReadable(data)
  stream.stderr = getReadable(error)
  setTimeout(() => stream.emit('close', isError ? 1 : 0), 0)
  return stream
}
```

then we could write line 79
```js
const stream = arrangeStream('irrelevant std out output', null, true)
```

Reply:
Yes, very good idea